docs/dev/libvirt/README.md

```diff
@@ -12,7 +12,7 @@ Before you begin, install the [build dependencies](../dependencies.md).
 Make sure you have KVM enabled by checking for the device:
 
 ```console
-$ ls -l /dev/kvm 
+$ ls -l /dev/kvm
```

Comment:
nitpick: irrelevant change to this commit. One commit should have one logical change only.

Reply:
I agree, this is just atom removing blanks

Reply:
yeah, still a separate logical change, even if tiny nobrainer. I'll leave it to you if you want to fix this or not. No biggie..

Reply:
Reverted.